apps/feedback/translations/src/en/pages.json

```diff
@@ -1,7 +1,6 @@
 {
   "start": {
     "header": "Give feedback",
-    "paragraph-1-hint": "(Limit is 1200 characters)",
```

Comment:
did you remove this because it's not relevant?

Reply:
Removing the redundant translation for the change above

Reply:
👍